tests/Mocks/MockCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Slim Framework (https://slimframework.com)
+ *
+ * @license https://github.com/slimphp/Slim-Console/blob/0.x/LICENSE.md (MIT License)
+ */
+
+declare(strict_types=1);
+
+namespace Slim\Tests\Console\Mocks;
+
+use Slim\Console\Command\AbstractCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class MockCommand extends AbstractCommand
+{
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'mock';
+
+    /**
+     * @var string
+     */
+    private $output = 'output';
+
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->writeln($this->getMockOutput());
+        return 1;
+    }
+
+    /**
+     * @return string
+     */
+    public function getMockOutput(): string
+    {
+        return $this->output;
+    }
+
+    /**
+     * @param string $output
+     * @return self
+     */
+    public function setMockOutput(string $output): self
```

Comment:
Is this method needed?

Reply:
I put it there in case we had to eventually do something special and change the output. I can remove it for now since it's unused!